plugins/generator-1.17.1/forge-1.17.1/templates/block/block.java.ftl

```diff
@@ -70,7 +70,11 @@ public class ${name}Block extends
 	</#if>
 
 	<#macro blockProperties>
+	<#if generator.map(data.colorOnMap, "mapcolors") != "DEFAULT">
+		BlockBehaviour.Properties.of(Material.${data.material},MaterialColor.${generator.map(data.colorOnMap, "mapcolors")})
+	<#else

```

Comment:
This else would need a > I would say, did you test this?

Reply:
Not this time 😬😣

Reply:
Fixed (and tested)